.github/ISSUE_TEMPLATE/ssw-rewards--sprint-forecast-template.md

```diff
@@ -13,30 +13,29 @@ assignees: ''
 [//]: # ( 3. Submit the Issue to send the Sprint Forecast )
 [//]: # (-------------------------------------------------)
 
-To: @adamcogan 
-Cc: @MattGoldmanSSW, @AndreasLengkeek
+To: @matt-goldman 
+Cc: @adamcogan, @JackDevAU
 
 (Checked by Matt)
 ### Hi Adam, 
 
 Current Sprint: | ✏️X
 -- | --
-Sprint Goal: | ✏️Content Migration
+Sprint Goal: | ✏️
 Sprint Duration: | ✏️1 week
 Project: | SSW.Rewards
 Project Portal: | https://ssw.visualstudio.com/SSW.Rewards
 Product Owner: | Adam Cogan
```

Comment:
Product owner has not been changed here

Reply:
But, you are not the product owner.

Do you still want this changed?

Reply:
No, leave it as is